common/elasticsearch/elasticsearch.go

```diff
@@ -127,8 +127,8 @@ func CreateElasticSearchService(uri *url.URL) (*ElasticSearchService, error) {
 	var startupFns []elastic.ClientOptionFunc
 	if len(opts["nodes"]) > 0 {
 		startupFns = append(startupFns, elastic.SetURL(opts["nodes"]...))
-	} else if uri.Opaque != "" {
-		startupFns = append(startupFns, elastic.SetURL(uri.Opaque))
+	} else if uri.Scheme != "" && uri.Host != "" {
```

Comment:
SGTM.

BTW, i think we need to document this behavior in the elasticsearch sink document.